README.md

```diff
@@ -1,2 +1,23 @@
 # hw01
 Repository for homework 1
+
+## Introduction
+
+#### Hello, my name is *Clara Sava-Segal*. This biography will consist of the following:
+  
+* Background
+* Course of Study and Interests
+* Plans post-graduation
+
+**Background:**
+I am currently a fourth year in the College at the University of Chicago, but I grew up right outside of [New York City](https://en.wikipedia.org/wiki/New_York_City). 
+
+**Course of Study and Interests:**
+As an undergrad, I've studied *Psychology* and *Comparative Human Development* and minored in *Biology.* I'm interested in the intersection between education and cognitive development and I worked in the [Casasanto Lab](http://www.casasanto.com), where I looked at the effects of different experiences on cognition. I am also currently working at a high school called Gary Comer College Prep.
+
+![Gary Comer College Prep Building.](https://github.com/csava1025/hw01/blob/master/GaryComer.jpg)
+
+**Plans post-graduation:**
+When I graduate, I will be moving to Palo Alto in California. 
+
```

Comment:
Evaluation HW-xx for Jane Doe
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      *     |              |            |
| **Coding strategy**         |     *      |              |            |
| **Presentation**            |      *     |              |            |
| **Achievement, creativity** |   *        |              |            |
| **Ease of access**          |  *         |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
N/A
* Some specific praise?
Overall this was a well executed assignment. It would have been great to also see a picture in this document but it is great the way it is!
* Something I learned?
N/A
* Specific constructive criticism?
N/A
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
N/A